### PR TITLE
RSL1g1

### PR DESCRIPTION
**Pull request by ricardopereira**

### CODE REVIEW

File: Spec/RestClientChannel.swift [hunk 1]
```diff
@@ -155,6 +155,39 @@ class RestClientChannel: QuickSpec {
...
+
+                        waitUntil(timeout: testTimeout) { done in
+                            let message = ARTMessage(name: nil, data: "message")
+                            expect(message.clientId).to(beNil())
+
+                            channel.publish([message]) { error in
+                                expect(error).to(beNil())
+                                channel.history { page, error in
+                                    expect(error).to(beNil())
+                                    expect(page!.items[0].clientId).to(equal("john"))
```
Comment (mattheworiordan) [inline Spec/RestClientChannel.swift]: This is right, but I think http://docs.ably.io/client-lib-development-guide/features/#RSL1g1b is wrong.  Do you agree @tcard that the spec is wrong?

Comment (tcard) [inline Spec/RestClientChannel.swift]: Yes, it seems in conflict with http://docs.ably.io/client-lib-development-guide/features/#RSA7a1.

Comment (ricardopereira) [inline Spec/RestClientChannel.swift]: Yes, `RSL1g1b` was confusing me.
Can I merge this?

**Comment by tcard** [comment]
LGTM

**Pull request merged**